invenio_app_ils/relations/api.py

```diff
@@ -27,6 +27,7 @@
     "invenio_app_ils.relations.nodes:PIDNodeRelated",
     "invenio_pidrelations.serializers.schemas.RelationSchema",
     "invenio_app_ils.relations.api.SiblingsRelation",
+    [],
```

Comment:
```suggestion
    None,
```
Setting mutable parameters is in general considered as bad practice, since we can change it on a runtime - especially in this case it is configuration so we don't want to allow for changes